CryptoAnalysis/src/main/java/crypto/HeadlessCryptoScanner.java

```diff
@@ -104,6 +110,22 @@ public static HeadlessCryptoScanner createFromOptions(String... args) {
 		} else {
 			callGraphAlogrithm = CG.CHA;
 		}
+		if(options.hasOption("ignorePackagesFilePath")) {
+			final File ignorePackageFile;
+			try {
+				ignorePackageFile = new File(options.getOptionValue("ignorePackagesFilePath"));
+				if(ignorePackageFile.exists()) {
+					ignorePackagesflag = true;
```

Comment:
enable the flag after `readAllLines()` has passed otherwise we might end up in a corrupt state when an `IOException` happens.

Reply:
enable the flag after readAllLines()